examples/data-fetching/src/pages/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import Layout from "../components/layout"
+
+const IndexPage = () => {
+  // ----------------------
+  // BUILD TIME DATA FETCHING
+  // ----------------------
+  const gatsbyRepoData = useStaticQuery(graphql`
+    query {
+      github {
+        repository(name: "gatsby", owner: "gatsbyjs") {
+          id
+          nameWithOwner
+          url
+        }
+      }
+    }
+  `)
+
+  // ----------------------
+  // RUNTIME DATA FETCHING
+  // ----------------------
+  const [starsCount, setStarsCount] = useState(0)
+  useEffect(() => {
+    const fetchData = async () => {
+      // get data from GitHub api
+      const result = await fetch(
+        `https://api.github.com/repos/gatsbyjs/gatsby`,
+        {
+          headers: {
+            Authorization: `Bearer ${process.env.GATSBY_GITHUB_TOKEN}`,
```

Comment:
This Part ist different to the examples in the docs and the receipt. 
Did we need to expose the GitHub token?

Reply:
oh yeah good catch this isn't necessary with the REST API! I'll remove that.